Example/Example Swift/ComponentAPIDocumentServiceProtocol.swift

```diff
@@ -21,30 +21,30 @@ enum CustomAnalysisError: GiniVisionError {
     }
 }
 
-enum Result<T> {
+enum CompletionResult<T> {
     case success(T)
     case failure(Error)
 }
 
-typealias UploadDocumentCompletion = (Result<GINIDocument>) -> Void
-typealias AnalysisCompletion = (Result<[String: Extraction]>) -> Void
+typealias ComponentAPIUploadDocumentCompletion = (CompletionResult<GINIDocument>) -> Void
+typealias ComponentAPIAnalysisCompletion = (CompletionResult<[String: Extraction]>) -> Void
 
-protocol DocumentServiceProtocol: class {
+protocol ComponentAPIDocumentServiceProtocol: class {
```

Comment:
Why shouldn't the document service protocol be the same for the component and screen API? What's the difference?

Reply:
I had to rename it because the compiler wasn't able to see that they were different and I was having problems with the test target.